### DISCUSSION TITLE
Donating the some `bitcoin-*` crates on crates.io.

### DISCUSSION THREAD

**Discussion by jeandudey** [rust-bitcoin/rust-bitcoin#1588] [Ideas]
Following the discussion on #550 and the possible crate-ification of some of the `rust-bitcoin` modules I can donate the `bitcoin-amount` crate name on crates.io:

- https://crates.io/crates/bitcoin-amount

I reserved it a while back when experimenting on `Amount` types a while back when I used to contribute here (hopefully I can contribute more this time :).

I think the `Amount` type is a very simple one and can be independent of almost all of bitcoin and doesn't depend on `std` and `alloc` stuff so it's ideal for using it as an inter-op type in other crates.

There's also these crates (@stevenroose  already owns these ones):

- https://crates.io/crates/bitcoin-rpc 
- https://crates.io/crates/bitcoin-rpc-json

**Comment by tcharding** [comment]
Awesome, thanks man. You motivated me to rework my amounts crate PR. Please feel free to jump in and suggest things on it since you have experience with the `Amount` type. I don't know the technical details of taking over the name on `crates.io` but @apoelstra is the release boss/slave at the moment :) Any contributions are super welcome, we are starting to get a few extra people involved this last year, its been great.

**Comment by tcharding** [comment]
I just realized we had plans to call it `bitcoin-units` not `bitcoin-amount` because things other than `Amount` might go in it e.g., `Target`.

**Reply by apoelstra** [reply]
Thanks! I couldn't remember where `Amount` fit into our crate-smashing plans :)
